cms/templates/partials/nhs_header.html

```diff
@@ -0,0 +1,35 @@
+{% load frontend_tags menu_tags %}
```

Comment:
Is this just a duplicate of the file in `partials/nhs_components/nhs_header.html`?